Functions/create_command.py

```diff
@@ -0,0 +1,39 @@
+import sys
+import json
+import os
+sys.path.append(os.path.join(str(os.getcwd()).replace("\\Functions", ''), "CommandsAndAdaptors"))
+from abstract_command import *
+from keys import Keys
+from errors import *
+
+
+class CreateCommand(AbstractCommand):
+    def __init__(self, schema_file):
+        self.data = None
+        self.path = None
+        self.schema_file = schema_file
+        self.schema_dir = Keys.SCHEMA_PATH
+        self.database_dir = Keys.SCHEMA_PATH
+        self.validate()
+        file = open(self.schema_file, 'r')
```

Comment:
Load can Throw exception here if the json is invalid.
I recommend load json inside private method and make try and catch when we call load json

Reply:
That will be checked in the validate function already